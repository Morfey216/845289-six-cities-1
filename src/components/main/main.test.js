import React from 'react';
import renderer from 'react-test-renderer';
import Main from './main';

it(`Main correctly renders`, () => {
  const tree = renderer
    .create(<Main offers={[]}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
