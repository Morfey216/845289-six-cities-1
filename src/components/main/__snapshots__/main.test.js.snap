// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main correctly renders 1`] = `
Array [
  <Header />,
  <main
    className="page__main page__main--index"
  >
    <h1
      className="visually-hidden"
    >
      Cities
    </h1>
    <div
      className="cities tabs"
    >
      <section
        className="locations container"
      >
        <ul
          className="locations__list tabs__list"
        >
          <li
            className="locations__item"
          >
            <a
              className="locations__item-link tabs__item tabs__item--active"
              onClick={[Function]}
            >
              <span>
                Paris
              </span>
            </a>
          </li>
          <li
            className="locations__item"
          >
            <a
              className="locations__item-link tabs__item "
              onClick={[Function]}
            >
              <span>
                Cologne
              </span>
            </a>
          </li>
        </ul>
      </section>
    </div>
    <div
      className="cities__places-wrapper"
    >
      <OffersEmpty
        currentCityName="Paris"
        currentOffersData={Array []}
      />
    </div>
  </main>,
]
`;
